openedx/core/djangoapps/user_api/api/profile.py

```diff
@@ -36,6 +37,10 @@ class ProfileInternalError(Exception):
     pass
 
 
+FULL_NAME_MAX_LENGTH = 255
```

Comment:
I deleted these variables, thinking they were only needed by the update_profile method that I deleted.

https://github.com/edx/edx-platform/pull/7123/files#diff-baf2090c93d7ed0e6ed4ead8a959b8a0L53